### PR TITLE
[docker-teamd]: Explicitly set LAG hwaddr

### PR DESCRIPTION
**Pull request by marian-pritsak**
Team device is initially created without any members and has a random HW
address, which is later changed to port's address. This configuration
sets team device's address explicitly to base MAC to avoid reassignment.

Signed-off-by: marian-pritsak <marianp@mellanox.com>

### CODE REVIEW

**Comment by lguohan** [comment]
test fails, can you check?

**Comment by lguohan** [comment]
also, I think centec also use your approach to set up the switch mac, can you check?

File: dockers/docker-teamd/start.sh [hunk 1]
```diff
@@ -5,8 +5,18 @@ TEAMD_CONF_PATH=/etc/teamd
 rm -rf $TEAMD_CONF_PATH
 mkdir -p $TEAMD_CONF_PATH
 
+SONIC_ASIC_TYPE=$(sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type)
+MAC_ADDRESS=$(ip link show eth0 | grep ether | awk '{print $2}')
+
+# Nellanox platform requires has last byte aligned
```
Comment (oleksandrivantsiv) [inline dockers/docker-teamd/start.sh]: Typo

Comment (marian-pritsak) [inline dockers/docker-teamd/start.sh]: Done

File: src/sonic-config-engine/tests/sample_output/PortChannel04.conf [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
     "device": "PortChannel04",
+    "hwaddr": "11:22:33:44:55:66",
```
Comment (lguohan) [inline src/sonic-config-engine/tests/sample_output/PortChannel04.conf]: 11 is multicast address, not a valid src mac address.

Comment (marian-pritsak) [inline src/sonic-config-engine/tests/sample_output/PortChannel04.conf]: Done

**Pull request merged**